test/unit/titles-get-delivery-days-test.js

```diff
@@ -0,0 +1,20 @@
+import { getDeliveryDays } from "../../lib/titles.js";
+
+describe("get delivery days", () => {
+  describe("get delivery days for dn", () => {
+    it("should return the correct days", () => {
+      const deliveryDates = getDeliveryDays("dn", "dn");
+      deliveryDates.should.eql([ "monday", "tuesday", "wednesday", "thursday", "friday", "saturday", "sunday" ]);
+    });
+  });
+
+  describe("get delivery days for title that does not have any deliverDays configured", () => {
+    it("should throw an error", () => {
+      try {
+        getDeliveryDays("paf", "paf");
```

Comment:
Shouldn't paf/paf have delivery days? Won't it break when we run their TOT sequence without them?

Maybe better to test with a digital product like `sportexpressen` (or a made up one)?